e2e/test/scenarios/sharing/alert/alert.cy.spec.js

```diff
@@ -92,6 +95,49 @@ describe("scenarios > alert", () => {
     });
   });
 
+  describe("with a webhook", () => {
```

Comment:
can you add the '@external` tag to this so we know it needs an external container?

Reply:
Good catch!